# Bioinfo-Project
It was a 10 day assignment given to me in course BSE322 Bioinformatics and Computational Biology

I was given a sequence and I have to perform the following tasks

1. Which protein does your sequence represent? Does it represent a full-length protein or a chain or domain of a protein? Use any tool of your choice to obtain this information. If you used a database search program, mention the name and show the alignment that you obtained from the program. (1 mark)

2. Provide the Refseq (NCBI reference sequence) and UniProt accession numbers corresponding to your protein, if available. Which organism is it found in?  (1 mark)

3. For your protein, show if there are any hits in the OMIM database (google this database!)? What do these results signify? (2 mark)

4. Find out the chromosome number and location on chromosome, where the gene encoding this protein is located. Give the identification/accession number of the gene from NCBI database. (2 marks)

5. Provide a pictorial view of the gene structure. How many introns and exons do you find? Which database did you find this information in? (2 marks)

6. From the NCBI or UniProt file, analyze the structural features of your protein. What secondary structures are present and at what positions? (2 marks)

7. BLAST your sequence to obtain two hit sequences (hit1 and hit2), one with high identity (preferably >80%) and another with low identity (E-value larger than 0.001). To achieve this, you can adjust the number of displayed sequences and/or choose a different database for your BLAST search. Show the two alignments and explain the various outputs like query coverage, identity, E-value given by BLAST for both the hits. (4 marks)

8. Prepare dot-plots of your sequence versus the two hit sequences obtained in the previous step using a dot-plot server, with default parameters (window/word size = 10), and show the obtained plots. “Dottup” from EMBOSS suite of programs is one such tool you may want to use for this purpose:
URL for Dottup: https://www.ebi.ac.uk/Tools/seqstats/emboss_dottup/
Change the word/window size to 3. Show the new images and explain the changes. (4 marks)

9. Use the same pairs of sequences (your sequence vs hit1 and your sequence vs hit2) and perform pairwise sequence alignments (local) using default parameters. You can use the WATER program on the EBI website for this. Show the alignments, and explain any differences from the alignments obtained using BLAST above. (4 marks)

10. Use the same program and perform the alignments for the same pairs of sequences using PAM250 scoring matrix. Show and explain any differences that you observe. (3 marks)

11. Now for the same pairs of sequences, use the default scoring matrix but change the default gap opening penalty to 1 and the default gap extension penalty to 1 and perform the two pairwise alignments again. Show and explain any changes. (3 marks)1. Which protein does your sequence represent? Does it represent a full-length protein or a chain or domain of a protein? Use any tool of your choice to obtain this information. If you used a database search program, mention the name and show the alignment that you obtained from the program. (1 mark)

